deployment/ansible/group_vars/all

```diff
@@ -1,5 +1,7 @@
 ---
 django_test_database: "{{ lookup('env', 'RF_TEST_DB_NAME') | default('test_rf', true) }}"
+sqs_queue: "{{ lookup('env', 'SQS_QUEUE_NAME') | default('TestQueue', true) }}"
```

Comment:
To support local development, I think that queue names will have to be namespaced by user names. Not sure how this will work until it is tested, but maybe prefix the queue name in `group_vars/development` with `lookup('env', 'USER')`. For `group_vars/test`, just `jenkins`. We'll probably only have one EC2 setup for now, so `group_vars/packer` can have no prefix.